.gitignore

```diff
@@ -15,4 +15,4 @@ Makefile*
 *.prl
 
 libconnman-qt/connman_*_interface.*
-
+connman-qt.pro.user
```

Comment:
I have added '*.pro.user' to my ~/.gitignore file solving this for all repositories. Nothing wrong with adding it here though.